index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <meta name="keywords" content="web design, frontend web development, software engineer, javascript engineer">
+        <title>Raphael Donanu | My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css">
+    </head>
+    <body>
+        <nav class="zone1">
+            <ul class="main-nav">
+                <li class="current"><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Me</a></li>
+                <li><a href="projects.html">My Projects</a></li>
+                <li class="push"><a href="contact.html">Contact Me</a></li>
```

Comment:
You may want to use a more descriptive class name such as `list-item` instead of `push`

Reply:
I have also modified this, thank you.